rules/__tests__/valid-expect.test.js

```diff
@@ -131,5 +139,39 @@ ruleTester.run('valid-expect', rules['valid-expect'], {
         },
       ],
     },
+    {
+      code:
+        'test("foo", () => { expect(Promise.resolve(2)).resolves.toBeDefined(); });',
+      errors: [{ message: "Must return or await 'expect.resolves' statement" }],
```

Comment:
can you assert on the locations as well?

Reply:
Yep, will make those changes 👍